core/ir/x86/instr_create_api.h

```diff
@@ -5039,6 +5039,26 @@
                            opnd_create_reg(DR_REG_EBX), opnd_create_reg(DR_REG_ECX), \
                            opnd_create_reg(DR_REG_EDX), opnd_create_reg(DR_REG_EAX), \
                            opnd_create_reg(DR_REG_ECX))
+
+/* 3 implicit destinations, 1 source */
```

Comment:
Can you please explain this further? If I'm not wrong, there seem to be 4 dests (eax, ebx, ecx, edx) and 4 srcs (same regs).

Reply:
I wasn't sure what to write there as I was thinking the same, but then the `cpuid` has the same comment, even though `ecx` can also be a source for `cpuid` (for subleafs). Thanks for confirming though, I will update that comment, but I think the `cpuid` one might need to be corrected too.

Reply:
Indeed, cpuid has 2 implicit sources as well, although ecx is leveraged only in specific cases.

Would you mind fixing the stale comment for cpuid? I think adding the change to this PR and mentioning it in the commit message would suffice, since it requires small modification.